packages/cli/package.json

```diff
@@ -68,7 +68,7 @@
         "@types/promptly": "^3.0.2",
         "babel-loader": "^9.1.2",
         "esbuild": "^0.17.19",
-        "vitest": "^1.4.0",
+        "vitest": "^0.33.0",
```

Comment:
I downgraded vitest because the main one is 0.33.0 and it will validate some types/methods but not run with it, which can lead to confusion